src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -216,7 +216,7 @@ public void visitJavadocToken(DetailNode ast) {
         final Optional<DetailNode> inlineTagNode = getInlineTagNode(ast);
         boolean shouldValidateUntaggedSummary = true;
         if (inlineTagNode.isPresent()) {
-            final DetailNode node = inlineTagNode.get();
+            final DetailNode node = inlineTagNode.orElseThrow();
             if (isSummaryTag(node) && isDefinedFirst(node)) {
```

Comment:
I am not really fun of this modern way to call `get()` , as we  have `isPresent` before.

@rdiachenko , @nrmancuso , what do you think on this style ?

Reply:
its idea suggestion too and the code literally makes just the same. Get interally makes a present check and throws as well, its just not that explicit.

Its very silly overall, yes i dont like it ether, but its suggestion from IDEA and other tool so its some kind of convention.

Reply:
This is actually the style recommended by the latest Java Optional API docs, so if we care about that, we should do it. I don’t mind either way.

Reply:
lets resolve this in dedicated branch. Reduced recipes to file for better overview.